### PR TITLE
tomboy: remove

### PR DESCRIPTION
**Pull request by dasj19**
###### Motivation for this change

When I tried to enable parallel builds as a part of https://github.com/NixOS/nixpkgs/issues/142338 I noticed that this package is broken in the latest nixpkgs unstable. So I checked upstream and saw that it is no longer developed. It was replaced by tomboy-ng

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -1030,6 +1030,7 @@ mapAliases ({
   tftp_hpa = tftp-hpa; # added 2015-04-03
   timescale-prometheus = promscale; # added 2020-09-29
   timetable = throw "timetable has been removed, as the upstream project has been abandoned"; # added 2021-09-05
+  tomboy = throw "tomboy is not actively developed anymore and was removed, request packaging tomboy-ng"; # added 2022-01-27
```
Comment (SuperSandro2000) [inline pkgs/top-level/aliases.nix:1033]:
```suggestion
  tomboy = throw "tomboy is not actively developed anymore and was removed"; # added 2022-01-27
```
it is very unlikely that this ever gets updated when tomboy-ng is introduced.

**Comment by dasj19** [comment]
@SuperSandro2000 thanks for the update

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by 06kellyjac** [comment]
I'm happy to nuke dead and broken packages from nixpkgs but I think it's important to at least ping the maintainers listed for that package

---

cc: @stesie

As a note tomboy-ng is actively maintained so if you want to add it back you could package that one:

https://wiki.gnome.org/Apps/Tomboy/tomboy-ng
https://github.com/tomboy-notes/tomboy-ng

**Pull request merged**